lib/justice_gov_sk.rb

```diff
@@ -0,0 +1,12 @@
+module JusticeGovSk
+  require 'active_support/all'
+  require 'active_job'
+  require 'nokogiri'
+
+  require 'justice_gov_sk/downloader'
```

Comment:
na toto prosim pouzime `ActiveSupport::Autoload` cez `#autoload` a `#autoload_at/under`, je to lepsie, vid http://api.rubyonrails.org/classes/ActiveSupport/Autoload.html